README.md

```diff
@@ -401,7 +409,123 @@ via [additional environment variables](https://github.com/FIWARE/helm-charts/blo
 
 ### IAM components
 
-> (similar to the Access Node section)
+The [DOME IAM-Framework](https://github.com/DOME-Marketplace/iam-components) is a set of microservices, that enables
+users in the DOME ecosystem to authenticate into the [DOME Marketplace](https://dome-marketplace.org).
+
+#### Overview and subcomponents
+
+The DOME IAM-Framework consists of multiple open-source components. The components are not required to be used, as long
+as alternatives providing the same interfaces are used.
+
+The IAM-Framework consists of following components:
+
+![IAM-components](./doc/img/iam.png)
+
+* The [Trusted Issuers List](https://github.com/fiware/trusted-issuers-list) service provides an [EBSI Trusted Issuers Registry](https://hub.ebsi.eu/apis/pilot/trusted-issuers-registry/v4) implementation to act as
+the Trusted-List-Service in the DSBA Trust and IAM Framework. In addition, a Trusted-Issuers-List API is provided to
+manage the issuers.
+* [VCVerifier](https://github.com/fiware/vcverifier) provides the necessary endpoints to offer SIOP-2/OIDC4VP compliant authentication flows. It exchanges
+VerifiableCredentials for JWT, that can be used for authorization and authentication in down-stream components.
+* [Credentials Config Service](https://github.com/fiware/credentials-config-service) manages and provides information about services and the credentials they are using.
+It returns the scope to be requested from the wallet per service. Furthermore, it specifies the credentials required and
+the issuers list endpoints to validate against, when checking access for a certain service.
+* The [Keycloak-VC-Issuer](https://github.com/fiware/keycloak-vc-issuer) is plugin for Keycloak to support SIOP-2/OIDC4VP clients and issue VerifiableCredentials through
+the OIDC4VCI-Protocol to compliant wallets.
+* [PDP](https://github.com/fiware/dsba-pdp) is an implementation of a Policy-Decision Point, evaluating Json-Web-Tokens containing VerifiableCredentials in a
+DSBA-compliant way. It also supports the evaluation in the context of i4Trust.
+* [Keyrock](https://github.com/ging/fiware-idm) is the FIWARE component responsible for Identity Management. Using Keyrock (in conjunction with other security
```

Comment:
Although these sentences are true, we don't use it as IdM component here. Basically what we use Keyrock for is only the iSHARE-compliant Authorization Registry part of it (we might add such link: https://dev.ishare.eu/delegation/endpoint.html), where attribute-based access policies are stored and used during the authorization progress. 
Later on, this will be replaced by an ODRL-compliant policy registry (a description of the policies will follow in the last section of the guide)